prql-compiler/prqlc/README.md

```diff
@@ -1,6 +1,7 @@
-# prqlc
+# PRQL compiler CLI
```

Comment:
Let's keep `prqlc` name here, it's the name of the cli after all.

Reply:
Sorry for the lack of explanation, `prqlc` is a great program name and I don't think it needs to be changed.

My point is to use "PRQL compiler CLI" as the package name that appears in winget manifests, etc.

For example, GitHub CLI (https://github.com/cli/cli) is a binary named `gh`, but the package name on the README and in the winget manifest is `GitHub CLI`.
https://github.com/microsoft/winget-pkgs/blob/03bf176f0ada12b0723f6d59f4b7ad6f5fba1630/manifests/g/GitHub/cli/2.29.0/GitHub.cli.locale.en-US.yaml#L2-L18

Reply:
Oh, sure - PRQL compiler CLI is a good description of what this is.